apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -14,6 +14,7 @@
 #    under the License.
 
 import copy
+import oslo_messaging
```

Comment:
oslo is a third party library, so it should be imported in the second section (below).
Our import convention is the following:

[standard libraries]
[third party]
[current project]